plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -291,14 +291,14 @@ public void testJoinPushdown()
                                 exchange(ExchangeNode.Scope.REMOTE, ExchangeNode.Type.REPLICATE,
                                         node(TableScanNode.class))));
 
+        Session session = joinPushdownEnabled(getSession());
+
         if (!hasBehavior(SUPPORTS_JOIN_PUSHDOWN)) {
-            assertThat(query("SELECT r.name, n.name FROM nation n JOIN region r ON n.regionkey = r.regionkey"))
+            assertThat(query(session, "SELECT r.name, n.name FROM nation n JOIN region r ON n.regionkey = r.regionkey"))
```

Comment:
i remember now!

initially `joinPushdownEnabled` did not have a usable, default implementation